aggregate/housing_production/hpd_housing_ny_affordable_housing.py

```diff
@@ -59,6 +59,14 @@ def load_housing_ny():
 
 
 def pivot_and_flatten_index(df, geography):
+    level_mapper = {
```

Comment:
@td928  Question.  To change `eli_units` to `units_eli` would you just need to adjust this snippet of code?  I was thinking about this last night and kicking myself because I think reversing units and the income band would better follow our convention.  If you'd hate me for making this change then don't worry about it.

Reply:
Not a problem at all! This should be an easy change